includes/active-directory-develop-guidedsetup-aspnetwebapp-test.md

```diff
@@ -92,6 +92,6 @@ You can restrict sign-in access to only user accounts that are in an Azure AD or
 
 #### Option 3: Use a custom method to validate issuers
 
-You can implement a custom method to validate issuers by using the **IssuerValidator** parameter. For more information about how to use this parameter, read about the [TokenValidationParameters class](https://msdn.microsoft.com/library/system.identitymodel.tokens.tokenvalidationparameters.aspx) on MSDN.
+You can implement a custom method to validate issuers by using the **IssuerValidator** parameter. For more information about how to use this parameter, read about the [TokenValidationParameters class](/previous-versions/visualstudio/dn464192(v=vs.114)) on MSDN.
```

Comment:
Since this page is no longer on MSDN, remove the "on MSDN" phrase as well.

```suggestion
You can implement a custom method to validate issuers by using the **IssuerValidator** parameter. For more information about how to use this parameter, read about the [TokenValidationParameters class](/previous-versions/visualstudio/dn464192(v=vs.114).
```